### DISCUSSION TITLE
On headings should we provide a UI to copy anchor links to the heading

### DISCUSSION THREAD

**Discussion by carolinejcourtney** [amnestywebsite/humanity-theme#66] [Enhancement Requests]
It would be helpful to provide a solution that allows front-end users to easily copy a jump link, or anchor link, so they can easily direct others to a specific section of pages on our website. 

This would be particularly relevant for long-form research. People might want to highlight a finding in a specific table or chapter in the report, but don't want to just send them to the top of the page and expect them to keep scrolling until the find the small bit of information they're looking for. 

We are already using these anchor links a lot, we have anchor links for tabs and can easily create them in the CMS. However, there is no visibility of these anchors for front end users, meaning its' just something that we as CMS users can use.

**Comment by AmnestyAM** [comment]
Interesting idea, you got any examples on other publishing platforms, would be interest to see their UI/UX.

**Comment by carolinejcourtney** [comment]
Wordpress.org documentation has it. 

https://wordpress.org/documentation/article/faq-i-make-changes-and-nothing-happens/

If you hover over the heading, a link icon appears which copies the anchor link to that heading.

**Reply by AmnestyAM** [reply]
is that the right link?

**Reply by carolinejcourtney** [reply]
Sorry, was just linking to an example, not documentation.  I'm talking about this feature. 
![image](https://github.com/amnestywebsite/humanity-theme/assets/26167906/263d7b68-09f3-4a43-beb6-5cfe01691313)

**Reply by carolinejcourtney** [reply]
If the user clicks the blue link icon next to the heading, the it copies the anchor link to that heading.

**Reply by AmnestyAM** [reply]
cool, it is here https://wordpress.org/documentation/article/heading-block/#advanced its in core so we will be supporting this soon

**Reply by AmnestyAM** [reply]
Looks like it already gives an ID to the heading, we would just need to either show a UI in the front-end when a class is added (to choose where we use this, if not everywhere), or if we want it everywhere show the UI everywhere and require nothing of the user.  
![image](https://github.com/amnestywebsite/humanity-theme/assets/37809410/0ab86399-92c3-4ea1-8390-26dcaac3e8b4)

The WP docs site on hover shows and icon, on click loads the URL with a little link to the left of the title presumably to indicate the user has been anchored to that part of the page.
